src/fitnesse/testsystems/slim/results/SlimExceptionResult.java

```diff
@@ -95,7 +95,11 @@ private String translateExceptionMessage(String exceptionMessage) {
       case COULD_NOT_INVOKE_CONSTRUCTOR:
         return "Could not invoke constructor for " + tokens[1];
       case NO_METHOD_IN_CLASS:
-        return String.format("Method %s not found in %s", tokens[1], tokens[2]);
+	if (tokens.length == 3){ // Legacy from Slim.Version <= 0.5 
```

Comment:
Can you elaborate on why this check is needed?

Reply:
This is old undocumented functionality. I added a new wiki testpage to explain why and improve documentation. See the commit
"added Unit test of SlimExceptionResult"